config/urls.py

```diff
@@ -5,8 +5,19 @@
 from django.views import defaults as default_views
 from django.views.generic import TemplateView
 
+
+class Home(TemplateView):
+    template_name = 'pages/home.html'
+
+    def get_context_data(self):
+        return {
+            'env_color': settings.ENV_COLOR,
```

Comment:
I like the flexibility implied by this setting. I am also wondering if this should/could be simplified to have a staging variant and a prod variant rather than using an actual color code. If we like the idea of setting new colors on more instances I am fine with this.

Reply:
Need local, staging and prod. I think that doing it this way is more aligned with the rest of how the options work instead of starting to define environment names.